active/ZEP0000.md

```diff
@@ -137,32 +164,51 @@ mentioned below. This allows the author to flesh out the draft ZEP to make it
 properly formatted, of high quality, and address initial concerns about the
 proposal.
 
-Spec ZEPs consist of two parts, a PR to the [zarr-specs] repository containing
-changes to the spec and a PR to the [ZEPs] repository containing a narrative
-document explaining the need, importance and use-case for the change. It is
-also highly recommended that the specification ZEP be accompanied by an
-implementation PR in at least one of the repositories represented in the
-[zarr-developers/zarr_implementations].
-
 ## Submitting a ZEP
 
-For SPEC ZEPs, the proposal should be submitted as a draft ZEP via two GitHub
-pull requests, one to the [ZEPs] repo and the other to the [zarr-specs] repository.
+### For SPEC ZEPs
+
+The proposal should be submitted as a draft ZEP via a GitHub pull
+request to the [zarr-specs] repository.
+
+The PR should contain the narrative document explaining the need, importance,
+and use case for the change of the ZEP and should be submitted in the
+zarr-specs repository with the name `zep-<n>: <title>.md` under
+[/zarr-specs/docs/zeps/draft], where `<n>` is an appropriately assigned
+four-digit number. The draft ZEP must use the [ZEP X - Template and
+Instructions][template.md] file.
+
+The SPEC ZEPs PR should also contain actual changes to the specification under
+[/zarr-specs/docs/specs/...] and mention the assigned four-digit ZEP number
+from the ZEP document.
+
+It is also highly recommended that the PR should accompany a complete
+implementation or at least an implementation PR in one of the repositories
+represented at [zarr.dev/implementations].
+
+### For Lean ZEPs
+
+The proposal should be submitted as a draft ZEP via a GitHub pull
+request to the [zarr-specs] repository.
+
+The PR should contain the specific changes to the Zarr specification without
+requiring a separate narrative document. As Lean ZEPs involve minor and
+non-controversial changes, a narrative document and the use of the template and
+instructions file are not necessary.
 
-The first PR should contain the narrative text of the ZEP and should be
-submitted in the zep repository with the name `zep-<n>.md` under [/zeps/draft/]
-where `<n>` is an appropriately assigned four-digit number. The draft ZEP must
-use the [ZEP X - Template and Instructions][template.md] file.
+The Lean ZEPs PR should be submitted in the [zarr-specs] repository with the
+name `<zep-<n>: <title>.md` under [/zarr-specs/docs/specs/...], where `<n>` is
+an appropriately assigned four-digit number.
```

Comment:
I am confused -- if a lean ZEP does not require a narrative document, what exactly is the `<zep - <n>: <title>.md` document? I thought a lean ZEP was just a PR against the spec, in which case shouldn't the procedure be "open a PR against the spec"?

Reply:
Fixed in https://github.com/zarr-developers/zeps/pull/59/commits/81b7ca2c9c554b77c0587ace192f0cbacb255a0b.

Reply:
summarizing last week's discussions including @d-v-b, @MSanKeys963 and me:
 * for the three types of changes:
   - ZEPs
   - "lean ZEP"
   - "just a PR"
 * we talked through some specific examples:
   - add a community codec (i.e. extension) -> just a PR, i.e. "patch"
   - move community codec to core codec -> ZEP (ZIC review), i.e. "minor"
   - choosing a default spelling (US vs UK) -> just a PR *but* might want a ZEP to record a decision
   - v4 -> include the world (ZSC, etc.)
 * we did not find a specific case for "lean ZEP" and instead decided it might be a *technical improvement* (template, bot, code, etc.) which helps expedite generating a ZEP from a simple PR.